### ISSUE TITLE
Hold request - Reason appears bolded briefly after holding the request

### ISSUE THREAD

**Issue by izarutskaya** [Expensify/App#40408]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.4.63-0
**Reproducible in staging?:** Y
**Reproducible in production?:** N
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Issue reported by:** Applause-Internal team

## Action Performed:
1. Go to staging.new.expensify.com
2. Create a manual request in the workspace chat.
3. Create a second request if there is only one request as Hold request option is not showing in the combined report.
4. Go to transaction thread.
5. Click 3-dot menu > Hold request.
6. Enter a reason and save it.

## Expected Result:
The reason in the transaction thread will not appear bolded.

## Actual Result:
The reason in the transaction thread is bolded briefly after holding the request.

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [x] Android: Native
- [x] Android: mWeb Chrome
- [x] iOS: Native
- [x] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [x] MacOS: Desktop

## Screenshots/Videos

https://github.com/Expensify/App/assets/115492554/38e9a200-ab6c-47da-b876-d28aacd57794

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by izarutskaya** [comment]
@trjExpensify I haven't added the External label as I wasn't 100% sure about this issue. Please take a look and add the label if you agree it's a bug and can be handled by external contributors.

**Comment by izarutskaya** [comment]
We think this issue might be related to the #collect project.

**Comment by shawnborton** [comment]
Yup, I agree that @GandalfGwaihir reported this issue first.

**Comment by techievivek** [comment]
Not overdue, on HOLD for https://github.com/Expensify/App/issues/37015

**Comment by trjExpensify** [comment]
PR still in review.

**Comment by techievivek** [comment]
PR on which we are holding this GH is still in review https://github.com/Expensify/App/pull/39452

**Comment by techievivek** [comment]
Still on hold for above PR.

**Comment by trjExpensify** [comment]
No change, still waiting on https://github.com/Expensify/App/pull/39452

**Comment by techievivek** [comment]
Bumped the PR, but we are still HOLDING this GH.

**Comment by trjExpensify** [comment]
Thanks, @techievivek!

**Comment by trjExpensify** [comment]
Asked what the next steps are in the PR we're held on.

**Comment by techievivek** [comment]
Still on HOLD, Tom has bumped the PR but we are still waiting for Robert to revert back.

**Comment by robertjchen** [comment]
Dropped some thoughts here: https://github.com/Expensify/App/pull/39452#issuecomment-2120465063 Let me know what you guys think

**Comment by trjExpensify** [comment]
Responded. Fixing the bold is the priority IMO, and markdown support for held reasons is secondary and polish.

**Comment by trjExpensify** [comment]
@paultsimura @robertjchen @techievivek thoughts on that?

**Comment by robertjchen** [comment]
I'm not sure, I came here from https://github.com/Expensify/App/issues/37015, so I'll defer to @paultsimura @techievivek on next steps here

**Comment by trjExpensify** [comment]
Yeah, looks like this got a bit muddled in the end. We have a PR up to fix bold now, so I don't think we should close it in the interest of #urgency. 

@GandalfGwaihir, I'll send you an offer for $125 for contributing to this issue. Sound good?

**Comment by trjExpensify** [comment]
>This PR https://github.com/Expensify/App/pull/39452 solved the issue of the bold style even before this issue started as mentioned here https://github.com/Expensify/App/issues/40408#issuecomment-2066738504

All good, we had two issues working against each other. It sometimes happens if it's not obvious they are interlinked. Let's move on. 👍 

In terms of next steps, here's what we should do: 

1. @dragnoir can you update the OP of [the PR](https://github.com/Expensify/App/pull/39452) to reflect what we're doing in it now (i.e only fixing bold, not adding markdown support for the held reason) and link it to this issue instead
2. I'll assign you to this issue to track the payment to completion for the fix, as well as @hoangzinh for the C+ review (@paultsimura, I'll drop you off this issue now as well, no need for two C+).
3. Let's close https://github.com/Expensify/App/issues/37015 as we don't plan to add markdown to the held reason for the time being
4. Once the regression period is over, I'll settle up with everyone accordingly. 

Sound good?

**Comment by trjExpensify** [comment]
Cool, I'll sort that out when I pay.

**Comment by trjExpensify** [comment]
Checklist time, please. :)

**Comment by robertjchen** [comment]
Awesome, thanks- I think we can pay and close this one out @trjExpensify ? 🙏
